### ISSUE TITLE
Border around images in UI

### ISSUE THREAD

**Issue by csxtan** [syl20bnr/spacemacs#13908]
#### Description
The components of the UI that are images have a small thin border around them, like the separators in the modeline and the heart in the "Made with <3 by the community" line, and the spacemacs logo.

#### Reproduction guide
Completely new emacs/spacemacs installation with the following configuration changes:
- layers: helm, auto-completion, emacs-lisp, git, markdown, org, syntax-checking
- default font set to 28, powerline-scale set to 0.85
- line numbers set to t
- default theme set to spacemacs-light
- powerline-default-separator set to arrow

*Observed behaviour*
Borders around all images
![screenshot-0](https://user-images.githubusercontent.com/20587609/92087171-0c871e80-ed80-11ea-890f-fa8b87a535e5.jpg)

*Expected behaviour*
No borders around images (especially in the mode line, so the separators mesh into the bar UI)

#### System Info :computer:
- OS: gnu/linux
- Emacs: 27.1
- Spacemacs: 0.200.13
- Spacemacs branch: master (rev. d46eacd83)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm auto-completion emacs-lisp git markdown org syntax-checking)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG CAIRO SOUND GPM DBUS GSETTINGS GLIB NOTIFY INOTIFY ACL GNUTLS LIBXML2 FREETYPE HARFBUZZ M17N_FLT LIBOTF ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS LIBSYSTEMD JSON PDUMPER LCMS2 GMP

**Comment by smile13241324** [comment]
@csxtan no idea where this comes from, I am on Manjaro with Gnome and I certainly doesn't have it. Please switch to develop and check whether the issue persists. If this does not solve your problem I can only think about some obscure rendering issues between emacs and your UI renderer, I had all kinds of these issues when I was still using XFCE.

If this does not solve your issue we need someone to dig into the home buffer rendering engine on a system where the issue is reproducible.

**Comment by csxtan** [comment]
I switched to develop and it didn't solve the problem. I also uninstalled spacemacs and this issue persists with vanilla emacs (the emacs logo gets the same border). I'm running Arch with awesome-wm (the problem is still there with i3 as well, I haven't tried any other window managers so maybe a DE would solve it).

Since it's probably not a spacemacs issue, I think this issue can be closed, but can anyone point me to somewhere more appropriate to ask for help on this issue?
